qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -411,18 +411,34 @@ class WebEngineHistory(browsertab.AbstractHistory):
     def current_idx(self):
         return self._history.currentItemIndex()
 
-    def back(self):
-        self._history.back()
+    def back(self, count):
```

Comment:
Since you are changing the definition of back here, maybe it should be changed in  [browsertab](https://github.com/iordanisg/qutebrowser/blob/c6ed4fe4f9160d28702a3e4c9cf7fa2d3a1809ca/qutebrowser/browser/browsertab.py#L468-L472) Las well?

Also, I _think_ that the back/forward logic can be moved up into AbstractHistory, I think that these implementations are the same for webkit/webengine/any future backends, especially if you are only calling `self.goToItem` in them. Not 100% sure if that will work though. (and the precedent says to keep it in the child as well).